digicampipe/io/event_stream.py

```diff
@@ -9,20 +10,17 @@ def event_stream(
     camera_geometry=None,
     expert_mode=None,
     max_events=None,
-    mc=False
+    source=None
```

Comment:
if source is not None then this function does not do anything I believe. we can remove source as arg.

Then latter we try to make guess_souce_from_path() smarter